client/components/StatUnitFormBody/FieldGroup.jsx

```diff
@@ -2,30 +2,12 @@ import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import { Form } from 'semantic-ui-react'
 
-class FieldGroup extends Component {
+export class FieldGroup extends Component {
   render() {
     const { isExtended, children } = this.props
     return (
       <Form.Group widths="equal">
-        {children[0].key == 'statId' && children}
```

Comment:
The removal of the key checking for the first child - why does that make sense?
What was the check for?

Reply:
This was a dirty hack to get something to show in the front-end. We don't need this hack anymore.